.circleci/config.yml

```
@@ -26,7 +26,12 @@ aliases:
   install_node_modules: &install_node_modules
     run:
       name: Install node modules
-      command: yarn --frozen-lockfile
+      command: yarn
+
+  check_lockfile: &check_lockfile
+    run:
+      name: Check for dirty lockfile
+      command: ./scripts/check-lockfile.sh || exit 1
```

probably can drop that ` || exit 1` part is script does have `else` case?
```suggestion
      command: ./scripts/check-lockfile.sh
```

feel free to ignore (doesn't matter much)

---

I don't think that would work without the exit, because it runs inside another script which won't have a non-zero exit otherwise